app/Enums/ContainerStatus.php

```diff
@@ -68,7 +68,7 @@ public function getColor(bool $hex = false): string
 
     public function getLabel(): string
     {
-        return str($this->value)->title();
+        return trans('strings.containerstatus.'.str($this->value));
```

Comment:
```suggestion
        return trans('strings.backupstatus.' . $this->value);
```
No need for str when not using headline since this Enum is a string
https://github.com/pelican-dev/panel/blob/de166bca0319f1240509f7927d2d26abb20a3b16/app/Enums/BackupStatus.php#L9